src/slack/views/postQuestionAnonymouslySubmitted.ts

```diff
@@ -22,17 +22,27 @@ export const postQuestionAnonymouslySubmitted: Middleware<SlackViewMiddlewareArg
 
     const text = `*_Someone has a question they'd like to ask!_* :thought_balloon: \n>${question}
 If you can answer this question, post a response in a thread!`;
+    ack();
 
     await app.client.chat.postMessage({
       token: env.slackToken,
       channel,
       text,
     });
-    ack();
+
+    await app.client.chat.postEphemeral({
```

Comment:
Can refactor the other uses in this file with the new pattern for using `client` from the method args?